aws/resource_aws_sqs_queue_policy.go

```diff
@@ -48,20 +52,20 @@ func resourceAwsSqsQueuePolicyUpsert(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error updating SQS attributes: %s", err)
 	}
 
-	d.SetId("sqs-policy-" + url)
```

Comment:
@radeksimko Do you recall why this was set as is?

Reply:
Hi @radeksimko, are you referring to something different than my state migration implementation?
```
MigrateState:  resourceAwsSqsQueuePolicyMigrateState,
SchemaVersion: 1,
```

If there's a better way to handle that, please let me know!

Reply:
doh 🤦‍♂️  sorry! I missed that one... In that case it's all good! Thanks.